dev/ScrollBar/TestUI/ScrollBarPage.xaml

```diff
@@ -0,0 +1,15 @@
+﻿<!-- Copyright (c) Microsoft Corporation. All rights reserved. Licensed under the MIT License. See LICENSE in the project root for license information. -->
+<local:TestPage
+    x:Class="MUXControlsTestApp.ScrollBarPage"
+    x:Name="ScrollBarTestPage"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:MUXControlsTestApp"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d">
+
+    <Grid Background="{ThemeResource ApplicationPageBackgroundThemeBrush}" Margin="12">
+        <ScrollBar/>
+    </Grid>
+</local:TestPage>
```

Comment:
Do we need an entire new page for this? Can we just use CornerRadiusPage?

Reply:
You can blame/thank me for this pattern. 😅
I think each control should have a landing page that shows that control in a couple different configurations. For manual or automated testing. So a bit of a different scenario than a big page with all controls for seeing CornerRadius side by side.
Of course given stuff going on on my end I didn't have much time to actually flesh out those pages other than to dump an instance of the control in each one. But that was the intent. Take the cost of creating a page now so that it was easy to iterate on in the future.

Reply:
Yeah we have CornerRadiusPage for rounded corner tests. I was thinking this was just a place holder page for other tests in the future, I guess it doesn't hurt to leave it there?